docs/shared/migration/migration-angular.md

```diff
@@ -339,20 +330,20 @@ ng e2e <app name>-e2e
 
 If you are using `Protractor` for E2E testing:
 
-- Delete the `apps/<app name>-e2e` folder that was generated to use Cypress.
-- Copy the `e2e` folder from your Angular CLI workspace into the `apps` folder.
+- Delete the `e2e` folder that was generated to use Cypress.
+- Copy the `e2e` folder from your Angular CLI workspace into the root folder.
 - Rename the `e2e` folder to `<app name>-e2e`.
-- Create the project configuration file at `apps/<app name>-e2e/project.json`.
-- Copy the project configuration for `app name` from the Angular CLI workspace `angular.json` file to `apps/<app name>-e2e/project.json` and adjust the file paths to be relative to `apps/<app name>-e2e`.
+- Create the project configuration file at `e2e/project.json`.
+- Copy the project configuration for `app name` from the Angular CLI workspace `angular.json` file to `e2e/project.json` and adjust the file paths to be relative to `e2e`.
 
-```json {% fileName="apps/<app name>-e2e/project.json" %}
+```json {% fileName="e2e/project.json" %}
```

Comment:
Again this should be more 1:1 now.

I would just mention to change `architect => targets` and `builder => executor`